### DISCUSSION TITLE
Allowed Amount File, PBM - Field Clarification

### DISCUSSION THREAD

**Discussion by djohnson071815** [CMSgov/price-transparency-guide#471] [Q&A] [answered]
As a PBM we currently do not have out of network contracts, therefore we cannot populate the following fields on the allowed amount file. Considering these fields are the only differentiation between this file and the in-network file, would this be required for us to populate the file as a whole on our website? If so, do we populate non applied fields a null value?
a.	allowed_amount
b.	billed_charge

**Comment by shaselton-usds** [comment, marked as answer]
@djohnson071815 Sorry for the confusion on my part, but can you clarify what you mean by "out of network contracts". The allowed amount file is meant to capture claims information that wasn't under a contract that is in-network. Are you saying that you have contracts with all provider networks and that you've never received a claim that was wasn't under contract?

**Reply by djohnson071815** [reply]
@shaselton-usds, To clarify we only process in-network claims.  We do not process claims for out of network pharmacies.  Any pharmacy that wants to process a claim would be required to contract with us.

**Reply by shaselton-usds** [reply]
If you don't have any out of network claims to populate the allowed amounts file, then you would create an "empty" file. [Here is an example of what that might look like.](https://github.com/CMSgov/price-transparency-guide/blob/59a3adc4f9d1ef6f86a306ea8714112db1805185/examples/allowed-amounts/allowed-amounts-single-plan-empty-sample.json)
